services/src/main/groovy/org/jfrog/artifactory/client/impl/FolderHandleImpl.groovy

```diff
@@ -0,0 +1,35 @@
+package org.jfrog.artifactory.client.impl
+
+import org.jfrog.artifactory.client.FolderHandle
+import org.jfrog.artifactory.client.model.FileList
+import org.jfrog.artifactory.client.model.impl.FileListImpl
+
+class FolderHandleImpl extends ItemHandleImpl implements FolderHandle {
```

Comment:
We're trying to gradually move away from groovy to java. Can you please move this class to be under the java directory structure?